src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,65 @@
-export const Tabs = () => {};
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react';
+
+interface Type {
+  tabsArr: {
+    id: string;
+    title: string;
+    content: string;
+  }[],
+
+  selectedTabId: string,
+
+  setTabId: (tabId: string)=> void,
+
+}
+
+export const Tabs: React.FC<Type> = ({
+  tabsArr,
+  selectedTabId,
+  setTabId,
+}) => {
+  const selecetedContent = tabsArr.find(tab => tab.id === selectedTabId);
+
+  let newContent = selecetedContent?.content;
+
+  if (newContent === undefined) {
+    newContent = tabsArr[0].content;
+  }
```

Comment:
you don't need this `if` as you've already set first tab as default in `App` component